RELEASE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 ## RELEASE NOTES
 
+### Version 2.79.15
```

Comment:
Will investigate after worked on other comments

Reply:
Spoke to Mo about this and he said 'it needs to be master version you referenced because the the pre-tagged is currently being tested'. Mo said they are using 'tagged pre-release version whilst the master candidate is being regression tested so for the devs its: 3.0.0-rc.1' but it's not relevant to the merge